### PR TITLE
Bug 1701154: Enable etcdHighNumberOfFailedGRPCRequests alerts

### PR DESCRIPTION
**Pull request by lilic**
This was fixed upstream, let's enable it downstream now as well. 🤞

### CODE REVIEW

**Comment by lilic** [comment]
/hold

We need a patch.

**Comment by hexfusion** [comment]
cc @lilic bummer :)

File: manifests/0000_90_etcd-operator_03_prometheusrule.yaml [hunk 1]
```diff
@@ -40,6 +40,35 @@ spec:
...
+          > 1
+      for: 10m
+      labels:
+        severity: warning
+    - alert: etcdHighNumberOfFailedGRPCRequests
+      annotations:
+        description: 'etcd cluster "{{ $labels.job }}": {{ $value }}% of requests
+          for {{ $labels.grpc_method }} failed on etcd instance {{ $labels.instance
+          }}.'
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/etcdHighNumberOfFailedGRPCRequests.md
```
Comment (lilic) [inline manifests/0000_90_etcd-operator_03_prometheusrule.yaml:62]: We still need to write this, will do it once the alert is merged.

**Comment by lilic** [comment]
Since we have etcd 3.5 now, we are ready to merge this. The only thing that fails is unrelated to this, its the PV test.


/hold cancel

@hexfusion please take a look again, thanks!

**Pull request merged**